Sources/Moya/MoyaProvider+Internal.swift

```diff
@@ -195,6 +197,16 @@ private extension MoyaProvider {
         return sendAlamofireRequest(alamoRequest, target: target, callbackQueue: callbackQueue, progress: progress, completion: completion)
     }
 
+    func sendUploadData(_ target: Target, request: URLRequest, callbackQueue: DispatchQueue?, data: Data, progress: ProgressBlock? = nil, completion: @escaping Completion) -> CancellableToken {
```

Comment:
If I see it correctly `sendUploadData` is exactly the same as `sendUploadFile` can we find some way to unify them in only one method? I think is ok to do it like this, but maybe would be good to have it unified :)

Reply:
Good idea, but I don't see any way to do this without more refactoring. I think it's ok like this for now.